ui/app/src/Router.tsx

```diff
@@ -39,6 +41,7 @@ function Router() {
           <Route path="/projects/:projectName" element={<GuardedProjectRoute />}>
             <Route path="" element={<ProjectView />} />
             <Route path=":tab" element={<ProjectView />} />
+            <Route path="explore" element={<ExploreView />} />
```

Comment:
Probably for a future PR: As it's working on Prometheus & Thanos UI (+ Grafana explore view), we may want to have the queries as part of the query params, so that you can share preconfigured explore views.

Reply:
Yeah!
That makes sense.